Include/internal/pycore_pystate.h

```diff
@@ -131,11 +135,13 @@ struct _gilstate_runtime_state {
 };
 
 /* hook for PyEval_GetFrame(), requested for Psyco */
-#define _PyThreadState_GetFrame _PyRuntime.gilstate.getframe
+#define _PyThreadState_GetFrame \
+    _PyRuntimeState_Get()->gilstate.getframe
```

Comment:
IMHO gilstate should be moved into PyInterpreterState.

Reply:
Also agreed.